test/camunda-cloud/ModelerSpec.js

```diff
@@ -108,12 +107,6 @@ describe('<CamundaCloudModeler>', function() {
       keyboard: {
         bindTo: document
       },
-      connectorsExtension: {
-        appendAnything: true
```

Comment:
Let us add the [`@bpmn-io/element-template-chooser`](https://github.com/bpmn-io/element-template-chooser) here so the examples remain usable (i.e. I can select element templates from the properties panel).